static/src/stylesheets/module/facia-garnett/_pillars.scss

```diff
@@ -161,4 +161,9 @@ $pillars: (
     .fc-item--pillar-#{$pillar} {
         @include colours($palette);
     }
+    .fc-sublink.fc-sublink--pillar-#{$pillar} {
```

Comment:
Is it possible to do this without specifying the `.fc-sublink` class in the selector? Or does it get overridden somewhere?

Reply:
@SiAdcock This is done